spaceship/lib/spaceship/connect_api/models/beta_group.rb

```diff
@@ -36,6 +36,15 @@ def self.type
       def post_bulk_beta_tester_assignments(beta_testers: nil)
         return Spaceship::ConnectAPI.post_bulk_beta_tester_assignments(beta_group_id: id, beta_testers: beta_testers)
       end
+
+      def delete!
+        return Spaceship::ConnectAPI.delete_beta_group(group_id: id)
+      end
+
+      def builds
```

Comment:
Are you cool if we rename this to `fetch_builds`? I like to the method name as an action if its doing an API request 😇 

```suggestion
      def fetch_builds
```